include/ktx.h

```diff
@@ -1281,15 +1287,20 @@ typedef struct ktxBasisParams {
              This will override the value chosen by @c qualityLevel.
          */
     char inputSwizzle[4];
-         /*!< A swizzle to apply before encoding. It must match the regular
+        /*!< A swizzle to apply before encoding. It must match the regular
              expression /^[rgba01]{4}$/. If both this and preSwizzle
              are specified ktxTexture_CompressBasisEx will raise
-             KTX_INVALID_OPERATION. */
-
+             KTX_INVALID_OPERATION.
+        */
     ktx_bool_t normalMap;
         /*!< Tunes codec parameters for better quality on normal maps (no
              selector RDO, no endpoint RDO). Only valid for linear textures.
          */
+    ktx_bool_t perceptual;
```

Comment:
Please remove this. The equivalent on the underlying encoder is set automatically for an sRGB texture.

The BasisU encoder's notion of perceptual is different from the ASTC encoder. Best to avoid possible confusion.